apps/api/src/app/shared/services/encryption/cipher.ts

```diff
@@ -0,0 +1,26 @@
+import { createCipheriv, createDecipheriv, randomBytes } from 'crypto';
+
+const ENCRYPTION_KEY = process.env.P_ENCRYPTION_KEY;
```

Comment:
```suggestion
const ENCRYPTION_KEY = process.env.STORE_ENCRYPTION_KEY;
```

Reply:
Haha agree i look at it today and did not like the name as well and changed to other one, will update to 'store' :)))